tests/unit/Entity/ItemIdTest.php

```diff
@@ -15,7 +16,7 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-class ItemIdTest extends \PHPUnit_Framework_TestCase {
+class ItemIdTest extends PHPUnit_Framework_TestCase {
```

Comment:
I saw Adrian doing this in more and more places. Personally I don't care that much.